azure-pipelines.yml

```diff
@@ -15,7 +15,6 @@ variables:
   SLDistributeDirectory: distribute
   SLFullDistributePath: $(SLBuildDirectory)\$(SLDistributeDirectory)
   BuildArtifact: obs-studio-node-us-$(Build.BuildId)-$(RuntimeName)-v$(ElectronVersion)
-  LibOBSVersion: 24.17.0
```

Comment:
Isnt removing of this line breaks our new release script?
https://github.com/stream-labs/slobs-backend-release/blob/727911e4cc2038113a9025df0a6f59cd50bee272/scripts/release.ts#L282

Reply:
Yes it does. Thank you for pointing that out.
I'll look into doing this differently then